scanners/nikto/parser/parser.test.js

```diff
@@ -34,3 +34,13 @@ test("parses OWASP Juice Shop result file into findings", async () => {
   await expect(validateParser(findings)).resolves.toBeUndefined();
   expect(findings).toMatchSnapshot();
 });
+
+test("parses no web server found finding correctly", async () => {
```

Comment:
```suggestion
test("parses 'no web server found' finding correctly", async () => {
```
I think this could improve readability